doc/release/1.7.2-notes.rst

```diff
@@ -5,16 +5,39 @@ SciPy 1.7.2 Release Notes
 .. contents::
 
 SciPy 1.7.2 is a bug-fix release with no new features
-compared to 1.7.1.
+compared to 1.7.1. Notably, the release includes wheels
+for Python 3.10, and wheels are now built with a newer
+version of OpenBLAS, 0.3.17.
 
 Authors
 =======
 
+* Ralf Gommers
+* Matt Haberland
+* Nicholas McKibben
+* Judah Rand +
+* Tyler Reddy
```

Comment:
Folks are helping drive some wheels repo changes--that should probably be taken into account in this list as well.